client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -3,427 +3,340 @@
-                                <li class="oui-tile__item">
-                                    <div data-ng-if="!hosting.isExpired"
-                                         data-ng-show="hosting.quotaPercent != null && hosting.quotaPercent >= 0">
-
-                                        <strong class="oui-tile__term" data-ng-bind-html="'hosting_dashboard_disk_usage' | translate"></strong>
+        <div class="alert alert-dismissible alert-info" role="alert"
+             data-ng-if="hosting.offer === 'KIMSUFI_2015'">
+            <button class="close" type="button" data-dismiss="alert"></button>
+            <span data-translate="hosting_dashboard_service_offer_KIMSUFI_2015_support"></span>
+        </div>
 
-                                            <div
-                                                class="oui-progress mt-2"
-                                                data-ng-class="{
-                                                    'oui-progress_success': hosting.quotaPercent <= 50,
-                                                    'oui-progress_warning': hosting.quotaPercent > 50 && hosting.quotaPercent <= 80,
-                                                    'oui-progress_error': hosting.quotaPercent > 80
-                                                }">
-                                                <div
-                                                    class="oui-progress__bar"
-                                                    role="progressbar"
-                                                    data-ng-class="{
-                                                        'oui-progress__bar_success': hosting.quotaPercent <= 50,
-                                                        'oui-progress__bar_warning': hosting.quotaPercent > 50 && hosting.quotaPercent <= 80,
-                                                        'oui-progress__bar_error': hosting.quotaPercent > 80
-                                                    }"
-                                                    data-ng-style="{'width': hosting.quotaPercent + '%'}"
-                                                    aria-valuenow="{{::hosting.quotaPercent}}"
-                                                    aria-valuemin="0"
-                                                    aria-valuemax="100">
-                                                    <span
-                                                        class="oui-progress__label"
-                                                        data-ng-bind="convertBytesSize(hosting.quotaUsed.value, hosting.quotaUsed.unit, 2) + ' / ' + convertBytesSize(hosting.quotaSize.value, hosting.quotaSize.unit)"
-                                                        data-ng-if="!hosting.isCloudWeb"></span>
-                                                </div>
-                                            </div>
+        <div class="row d-md-flex" data-ng-if="!hosting.isExpired">
```

Comment:
https://github.com/twbs/bootlint/wiki/E013, `row` element cannot be direct child of another `row` 
(See line 5)